CHANGES.md

```diff
@@ -1,5 +1,19 @@
 # Changes
 
+## edge-21.6.3
+
+This release moves the Linkerd proxy to a more minimal Docker base image,
+adds a check for detecting certain network misconfigurations, and replaces
+the deprecated OpenCensus collector with the OpenTelemetry collecting in the
```

Comment:
nit: wasn't sure if this is a typo, both forms make sense to me (i.e collector and collecting) but thought I'd highlight it to see what you think. 

```suggestion
the deprecated OpenCensus collector with the OpenTelemetry collector in the
```